modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs

```diff
@@ -1197,6 +1219,38 @@ public static double MoveToward(double from, double to, double delta)
             return from + (Math.Sign(to - from) * delta);
         }
 
+        /// <summary>
+        /// Moves <paramref name="from"/> toward <paramref name="to"/>
+        /// by the <paramref name="delta"/> angle (in radians).
+        /// Use a negative <paramref name="delta"/> angle to move away.
+        /// Similar to <see cref="MoveToward"/> but wraps around <see cref="Tau"/> and creates a shortest path.
+        /// </summary>
+        /// <param name="from">The start Angle.</param>
+        /// <param name="to">The Angle to move towards.</param>
+        /// <param name="delta">The amount to move by.</param>
+        /// <returns>The Angle after moving.</returns>
+        public static float MoveTowardAngle(float from, float to, float delta)
+        {
+            float diff = AngleDifference(from, to);
+            return from + Min(Abs(diff), delta) * Sign(diff);
```

Comment:
It looks like other methods use `Math.Abs` and so on, is that the standard approach here?

Reply:
Fixed!